test/vector_search_validator/test_validator.py

```diff
@@ -0,0 +1,82 @@
+#
+# Copyright (C) 2025-present ScyllaDB
+#
+# SPDX-License-Identifier: LicenseRef-ScyllaDB-Source-Available-1.0
+#
+
+import os
+from pathlib import Path
+import pytest
+import subprocess
+from test import BUILD_DIR
+
+# IP address for custom DNS server inside linux namespace
+DNS_IP = "127.0.1.1"
+# 3 tuple IP address as a base for scylla or vector-store nodes
+BASE_IP = "127.0.2"
```

Comment:
This is a bad idea to hardcode the IP. We already had several issues with hardcoded IP. Please use [HostRegistry](https://github.com/scylladb/scylladb/blob/eb5e9f728cf1a6e829164257408ed75b4635a025/test/pylib/host_registry.py#L17C7-L17C19) to obtain the IP address.